client/src/pages/Main.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from "react";
+import Header from "../Components/Header";
+import Mention from "../Components/Mention";
+import Grid from "@material-ui/core/Grid";
+import SwitchSelector from "react-switch-selector";
+import { withStyles } from "@material-ui/core/styles";
+
+const styles = (theme) => ({
+  typography: {
+    fontFamily: theme.typography.fontFamily,
+  },
+  palette: {
+    background: theme.palette.primary.main,
+  },
+});
+
+class Main extends Component {
+  render() {
+    const { classes } = this.props;
+
```

Comment:
Notice that the background in this page is actually a light shade of blue. You can use a color picker to get the exact tone rgb